src/stories/Page.stories.js

```diff
@@ -1,4 +1,5 @@
-import { within, userEvent } from '@storybook/testing-library'
+import { userEvent, within } from '@storybook/testing-library'
```

Comment:
Change the import order for this file as the previous one, and remove the empty line below it.